services/ui-auth/libs/users.js

```diff
@@ -1,4 +1,4 @@
-[
```

Comment:
We're keeping this as users.json in the other apps, right?

Reply:
Yep, my B!